src/protocols/MeterS0.cpp

```diff
@@ -238,31 +238,34 @@ void MeterS0::counter_thread()
 		if (is_blocking) {
 			bool timeout = false;
 			if (_hwif->waitForImpulse( timeout )) {
+				//Event erkannt. Wert des Events ist, aufgrund des prellen reiner Zufall!!
+				//deswegen erst entprellen .....
```

Comment:
da der Wert des Events nicht genutzt wird (nur der des dir_pins), verstehe ich immer noch nicht, warum der Patch das Problem verbessern sollte.
Einziger Unterschied ist, dass unten die Prüfung auf den Wert eingebaut wird.
Meine Logik war: wenn es an der Leitung "zappelt", dann wird das als Event gezählt und danach aber die Debounce Zeit abgewartet. D.h. Zappeln innerhalb der Debounce Zeit wird ignoriert.
Du baust das jetzt um in:
- nach Zappeln erst debounce Zeit warten und dann nur zählen, wenn der Wert 1 ist (bis auf die fehlende Prüfung bei hwif_dir).